.github/workflows/ci-code.yml

```diff
@@ -64,16 +64,22 @@ jobs:
         python-version: ${{ matrix.python-version }}
 
     - name: Setup environment
-      # Note: The virtual environment in .venv was created by uv in previous step
-      run: source .venv/bin/activate && .github/workflows/setup.sh
+      run: .github/workflows/setup.sh
 
     - name: Run test suite
       env:
         AIIDA_TEST_PROFILE: test_aiida
         AIIDA_WARN_v3: 1
-      # Python 3.12 has a performance regression when running with code coverage
+      # NOTE1: Python 3.12 has a performance regression when running with code coverage
       # so run code coverage only for python 3.9.
-      run: uv run pytest -n auto --db-backend psql -m 'not nightly' tests/ ${{ matrix.python-version == '3.9' && '--cov aiida' || '' }}
+      # NOTE2: Unset CI envvar to workaround a pymatgen issue for Python 3.9
+      # https://github.com/materialsproject/pymatgen/issues/4243
+      # TODO: Remove a workaround for VIRTUAL_ENV once the setup-uv action is updated
```

Comment:
This issue is described in detail in https://github.com/aiidateam/aiida-core/pull/6676#issuecomment-2565971534.

I'll remove the workaround hopefully soon in a subsequent PR.